lego/apps/polls/tests/test_api_polls.py

```diff
@@ -46,55 +46,53 @@ def setUp(self):
 
     def test_create_poll_authenticated(self):
         """A user with permissions should be able to create a poll"""
-        self.client.force_authenticate(self.authenticated_user)
+        self.client.force_login(self.authenticated_user)
```

Comment:
Is there a reason to change this to `force_login`? The "django-rest-framework" way of doing it is `force_authenticated`, while `force_login` is from the base django test case.